src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -180,4 +180,13 @@
   <data name="Interactive_host_process_platform" xml:space="preserve">
     <value>Interactive host process platform</value>
   </data>
+  <data name="Click_to_select_member" xml:space="preserve">
+    <value>Click to select member</value>
+  </data>
```

Comment:
where do these resources show up?  can you show a picture?

Reply:
Yup I will post picture when its ready

Reply:
![image](https://user-images.githubusercontent.com/24360909/112686776-bc62cd00-8e33-11eb-81d9-0b6cbe36776a.png)

Reply:
my preference would be something a bit richer.  something more like:

```
Implements:
    interface Blah.Sub()
    overriden by Blorp.Sub()

Click to navigate or see more information.
```

Basically, i'm not a fan of a tooltip saying i can click on something.

Note how debug breakpoints work:

![image](https://user-images.githubusercontent.com/4564579/112691367-6f362980-8e3a-11eb-8136-68260057e16c.png)

It provides info, and doesn't try to tell you what clicking does.

Reply:
Got you. this is meaningful for the tooltip of an inheritance target (I think we could show  'location' -> 'file' -> 'class' -> 'Project' info for them.) I'll modify that.
For the case I posted, (e1 and e2 are both inherited), do you feel 'Click to select member' is meaningful? Or maybe we should show "event e1, event e2 have inheritance target?"

Reply:
Can we show what was originally planned for the tooltip? Text giving richer information and a tip on how to navigate:

![image](https://user-images.githubusercontent.com/46729679/112885081-2625e580-9085-11eb-8c31-32310bb362e2.png)

Reply:
does our UI generally have to state that it is clickable?  That seems like it will get old fast.  THis is what quick-actions shows:

![image](https://user-images.githubusercontent.com/4564579/112910959-423b7e00-90a9-11eb-8c3d-36a2a4d22cff.png)

Can we just follow that style?

Reply:
(this is actually, imo, how the entire feature should look and feel).  It's effectively in teh same location, with just a different glyph chosen by us.  if you open it (by mouse or keyboard) you should get a list of items, just like quick-actions do.  We should not have to say it's clickable, that should be inferred just by having it look and feel just like our other margin adornments.

Reply:
I think it requires adding a button and an arrow to the background of the button. I can add that

Reply:
I will follow up with UX on this. In some cases selecting the icon can automatically navigate you thought right? Or is the behavior to show you the list of results even if there is only one?

Reply:
@mikadumont The current implementation will show the results even if there is only one

Reply:
![image](https://user-images.githubusercontent.com/24360909/113038025-dad60a80-914a-11eb-97f7-6d019247e10c.png)
@CyrusNajmabadi is this the effect you want?

Reply:
Well, I think we have three problems discussed here so far,
I'll try to summarize all the existing problem here.
1. Tooltip. (As Mika posted, the solution is to use "'bar' is inherited (click or press <short-cut>)") Because we have no short cut right now, so I temporary remove the short-cut part. Later it should be added when we have the short-cut.
2. Icon. Do  we need a margin that looks like 'clickable' for initial release? 
3. Automatically do the navigation if there is only one target.  (Synced with Mika offline, adding telemetry to tell us whether users are just using the margin to view targets or navigate)

tag @CyrusNajmabadi @mikadumont  @sharwell  @ryzngard  for opinions

Reply:
> Icon. Do we need a margin that looks like 'clickable' for initial release?

I would think so, yes.  it's the VS way.

> Automatically do the navigation if there is only one target. (Synced with Mika offline, adding telemetry to tell us whether users are just using the margin to view targets or navigate)

Definitely interesting.  there are two metaphors i care about:

1. finding out what something points to
2. navigating to that.

So bar of me felt that click shouldn't automatically take you there.  Howeve,r if i can hover and see the info without needing to navigate, i think that may be good enough.

Note: we really need to calssify our text here.  it just looks weird to have this be all gray text.

Reply:
Updates for this thread:
1. Margin effect
![inheritanceMargin8](https://user-images.githubusercontent.com/24360909/113217203-6d9da480-9232-11eb-97e2-5cc2d001326e.gif)

2. Classified tooltip
![image](https://user-images.githubusercontent.com/24360909/113217263-860dbf00-9232-11eb-8a9a-457090bfc3ea.png)

3. Telemetry
 I have added two telemeties reporting
    1. When navigation is performed
    2. When the targets context menu is open.